config/v1/types_project.go

```diff
@@ -20,8 +19,41 @@ type Project struct {
 }
 
 type ProjectSpec struct {
-	// project request message
-	// project request template
+	// DefaultNodeSelector holds default project node label selector
+	DefaultNodeSelector string `json:"defaultNodeSelector"`
+
+	// ProjectRequestMessage is the string presented to a user if they are unable to request a project via the projectrequest api endpoint
+	ProjectRequestMessage string `json:"projectRequestMessage"`
+
+	// ProjectRequestTemplate is the template to use for creating projects in response to projectrequest.
+	// It is in the format namespace/template and it is optional.
+	// If it is not specified, a default template is used.
+	ProjectRequestTemplate string `json:"projectRequestTemplate"`
+
+	// SecurityAllocator controls the automatic allocation of UIDs and MCS labels to a project. If nil, allocation is disabled.
+	SecurityAllocator *SecurityAllocator `json:"securityAllocator,omitempty"`
+}
+
+type SecurityAllocator struct {
```

Comment:
remove from  this pull.  If @openshift/sig-auth needs it, they can decide how to structure it.